build_modules/lib/src/kernel_builder.dart

```diff
@@ -74,11 +74,15 @@ class KernelBuilder implements Builder {
   /// Experiments to pass to kernel (as --enable-experiment=<experiment> args).
   final Iterable<String> experiments;
 
+  /// Whether or not strong null safety should be enabled.
```

Comment:
[nit]
```suggestion
  /// Whether or not sound null safety should be enabled.
```

Can we expand the docs while we are at it? Maybe we should say something about how we need to always use sound kernel files as inputs to build sound outputs, and vice versa?